cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1110,6 +1110,12 @@
     public static final String CFG_CDAP_MASTER_KRB_PRINCIPAL = "cdap.master.kerberos.principal";
 
     public static final String UGI_CACHE_EXPIRATION_MS = "cdap.ugi.cache.expiration.ms";
+    /** Case conversion type */
+    public static final String SECURITY_AUTH_USERNAME_CASE_CONVERSION_TYPE =
```

Comment:
Please move the constant under the `AuthenticationServer` inner class and no need to have `SECURITY_AUTH_` prefix in the  constant.

Reply:
done.
Sorry for the delay, don't know why I didn't get any notification for your review comments.